cli/cmd/turbo/main.go

```diff
@@ -19,6 +22,19 @@ func nativeRunWithArgs(argc C.int, argv **C.char) C.uint {
 	for i, arg := range unsafe.Slice(argv, arglen) {
 		args[i] = C.GoString(arg)
 	}
+
 	exitCode := cmd.RunWithArgs(args, turboVersion)
 	return C.uint(exitCode)
 }
+
+//export nativeRunWithTurboState
+func nativeRunWithTurboState(turboStateString string) C.uint {
+	turboState := turbostate.CLIExecutionStateFromRust{}
+	err := json.Unmarshal([]byte(turboStateString), &turboState)
+	if err != nil {
+		fmt.Printf("Error unmarshalling turboState: %v\n", err)
```

Comment:
Yeah this is a larger question I've been wondering about: what's our philosophy for logging out errors? Because it seems like we don't log out much when we hit an error.